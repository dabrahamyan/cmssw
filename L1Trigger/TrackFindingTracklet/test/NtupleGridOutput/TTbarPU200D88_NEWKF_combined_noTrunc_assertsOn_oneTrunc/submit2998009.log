000 (2998009.000.000) 09/12 10:55:24 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_159281_1896>
...
040 (2998009.000.000) 09/12 12:45:50 Started transferring input files
	Transferring to host: <128.142.137.55:9618?addrs=128.142.137.55-9618+[2001-1458-301-5e--100-2b]-9618&alias=b7g19p7029.cern.ch&noUDP&sock=slot1_45_3090_ef37_4391>
...
040 (2998009.000.000) 09/12 12:45:50 Finished transferring input files
...
001 (2998009.000.000) 09/12 12:45:51 Job executing on host: <128.142.137.55:9618?addrs=128.142.137.55-9618+[2001-1458-301-5e--100-2b]-9618&alias=b7g19p7029.cern.ch&noUDP&sock=startd_2841_44a6>
...
006 (2998009.000.000) 09/12 12:46:01 Image size of job updated: 266888
	32  -  MemoryUsage of job (MB)
	32372  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 12:51:01 Image size of job updated: 2427288
	1010  -  MemoryUsage of job (MB)
	1034172  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 12:56:01 Image size of job updated: 2427288
	1020  -  MemoryUsage of job (MB)
	1043832  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 13:11:03 Image size of job updated: 2435888
	1020  -  MemoryUsage of job (MB)
	1043832  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 13:41:07 Image size of job updated: 2435888
	1020  -  MemoryUsage of job (MB)
	1044208  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 13:46:09 Image size of job updated: 2435888
	1027  -  MemoryUsage of job (MB)
	1050880  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 13:56:09 Image size of job updated: 2599728
	1027  -  MemoryUsage of job (MB)
	1050880  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 14:16:12 Image size of job updated: 2599756
	1036  -  MemoryUsage of job (MB)
	1060104  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 14:26:13 Image size of job updated: 6089544
	1330  -  MemoryUsage of job (MB)
	1361108  -  ResidentSetSize of job (KB)
...
006 (2998009.000.000) 09/12 14:26:33 Image size of job updated: 6376752
	1330  -  MemoryUsage of job (MB)
	1361192  -  ResidentSetSize of job (KB)
...
040 (2998009.000.000) 09/12 14:26:34 Started transferring output files
...
040 (2998009.000.000) 09/12 14:26:34 Finished transferring output files
...
005 (2998009.000.000) 09/12 14:26:34 Job terminated.
	(1) Normal termination (return value 139)
		Usr 0 01:31:28, Sys 0 00:02:20  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:31:28, Sys 0 00:02:20  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	99067  -  Run Bytes Sent By Job
	699  -  Run Bytes Received By Job
	99067  -  Total Bytes Sent By Job
	699  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.99        1         1 
	   Disk (KB)            :   118           1   1547495 
	   Memory (MB)          :  1330        2000      2000 

	Job terminated of its own accord at 2023-09-12T12:26:33Z.
...
