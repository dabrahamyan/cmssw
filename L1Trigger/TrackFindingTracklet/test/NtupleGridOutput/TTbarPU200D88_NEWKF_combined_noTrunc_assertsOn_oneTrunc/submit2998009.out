using geometry D88 (tilted)
Tue Sep 12 14:26:01 CEST 2023
Thread 7 (Thread 0x2b4fe5036700 (LWP 184) "cmsRun"):
#0  0x00002b4fb1511b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4fb1511bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4fb1511c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4fb7870576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4fb7870639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b4fe4e35700 (LWP 183) "cmsRun"):
#0  0x00002b4fb1511b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4fb1511bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4fb1511c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4fb7870576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4fb7870639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b4fe4c34700 (LWP 182) "cmsRun"):
#0  0x00002b4fb1511b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4fb1511bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4fb1511c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4fb7870576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4fb7870639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b4fe4a33700 (LWP 181) "cmsRun"):
#0  0x00002b4fb1512e9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b4fb75ad10f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b4fb77de2d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002b4fb77de429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b4fe4832700 (LWP 180) "cmsRun"):
#0  0x00002b4fb181f0e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b4fb75a75c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b4fb75a3d4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002b4fb75ac818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b4fd5f5d700 (LWP 173) "cmsRun"):
#0  0x00002b4fb15131d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b4fb7f02ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b4fb7f036ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b4fb10e7f90 in std::execute_native_thread_routine (__p=0x2b4fc05bf060) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b4fb150bea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4fb181eb0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b4fb38a1e40 (LWP 155) "cmsRun"):
#0  0x00002b4fb1813ddd in poll () from /lib64/libc.so.6
#1  0x00002b4fb7f02dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b4fb7f0379c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b4fb7f0624b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b4fe22763b5 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /afs/cern.ch/user/d/dabraham/private/test/tempCMSSWs/one/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#6  0x00002b4fe22a7386 in trklet::ProducerKFout::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/tempCMSSWs/one/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#7  0x00002b4faf1341e3 in edm::stream::EDProducerAdaptorBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#8  0x00002b4faf116c3f in edm::WorkerT<edm::stream::EDProducerAdaptorBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#9  0x00002b4faf066b25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#10 0x00002b4faf066e13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#11 0x00002b4faf069546 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#12 0x00002b4faee02879 in tbb::detail::d1::function_task<edm::WaitingTaskList::announce()::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#13 0x00002b4fb08fcbec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2b4fb503a300, this=0x2b4fb5067700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#14 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2b4fb5067700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#15 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#16 0x00002b4faefe122d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002b4faefc9310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002b4faefd5da1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#20 0x00002b4fb08eb0eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#21 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#22 0x000000000040971c in main ()
